ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -254,10 +254,7 @@ def LeftShift(*, x, y, name="LeftShift"):
     return ivy.bitwise_left_shift(x, y)
 
 
-@to_ivy_arrays_and_back
-def Less(*, x, y, name="Less"):
-    x, y = check_tensorflow_casting(x, y)
-    return ivy.less(x, y)
+Less = to_ivy_arrays_and_back(map_raw_ops_alias(tf_frontend.math.less))
```

Comment:
You are not supposed to make changes to that file. Please remove your edit from here.

Reply:
Resolved it @kurshakuz